From f6d7c30415898c73a3b1c207d2acb496f53f67ee Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Thu, 30 Oct 2014 09:36:49 +0100
Subject: [PATCH] Decrease the tested memory limits because of failures on arm

Related to #42
---
 test/Test.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/Test.java b/test/Test.java
index cd4acf4..cbbc113 100644
--- a/test/Test.java
+++ b/test/Test.java
@@ -71,8 +71,8 @@ public class Test {
         { int[]      intArray1D = new int[DIM_SIZE * DIM_SIZE]; }
         /* allocated size = 4096*4096*4 bytes */
         { int[][]    intArray2D = new int[DIM_SIZE][DIM_SIZE]; }
-        { double[]   doubleArray1D = new double[DIM_SIZE * DIM_SIZE]; }
-        { double[][] doubleArray2D = new double[DIM_SIZE][DIM_SIZE]; }
+        { double[]   doubleArray1D = new double[DIM_SIZE * (DIM_SIZE/2)]; }
+        { double[][] doubleArray2D = new double[DIM_SIZE][DIM_SIZE/2]; }
 
         // string could be allocated using some base array
         { String s = new String(new byte[DIM_SIZE * DIM_SIZE]); }
-- 
2.1.0

